### PR TITLE
Improve `stressGraphics` code, support web mode

### PR DESCRIPTION
**Pull request by linev**
* Make central function `TestReport()` which executes all planned tests for the canvas. 
* Print report when all images and scripts are generated. 
* Resort function orders to avoid need of forward declarations
* Use `TString::Format()` instead of `snprintf`
* Use `TFile::Open()` instead of `new TFile`
* Improve handling of command line args like `-k` or `-v`
* Create SVG instead of PS files when running in web mode, provide correspondent ref file
* Adjust web ref file to work on Chrome and Firefox

All this simplify structure of sressGraphics code.

Execution time in web mode with chrome browser 109s.

```
./stressGraphics --web=chrome
...

**********************************************************************
*  All the tests passed. :-)
**********************************************************************
*  SYS: Linux hades58 6.10.11-1-default #1 SMP PREEMPT_DYNAMIC Thu S
**********************************************************************
* stressGraphics: Real Time = 109.74 seconds Cpu Time =   8.32 seconds
**********************************************************************
*  ROOTMARKS =4870.6   *  Root 6.33.01   20231010/0
**********************************************************************
```

### CODE REVIEW

**Comment by couet** [comment]
Why change stressGraphics ? It is meant to test the old graphics and does it for years the proper way. It is a standalone program ... I don't understand the point.

**Comment by linev** [comment]
All tests remain exactly the same - just invocation order at the end little bit changed.

**Comment by couet** [comment]
Does it run as before ?

**Comment by couet** [comment]
There is 2 different ref file

**Comment by linev** [comment]
> Does it run as before ?

Exactly the same. No single change in refs are required.

**Comment by couet** [comment]
note that roottest use it also

**Comment by linev** [comment]
> note that roottest use it also

of course. Therefore if there are some problems - we should see them in CI

**Comment by couet** [comment]
let wait the ci run

File: test/stressGraphics.cxx [hunk 1]
```diff
@@ -81,390 +81,56 @@
...
+Int_t     gPNGRefNb[kMaxNumTests];
+Int_t     gPNGErrNb[kMaxNumTests];
+Int_t     gPS2RefNb[kMaxNumTests];
+Int_t     gPS2ErrNb[kMaxNumTests];
+Bool_t    gWebMode = kFALSE;
+Bool_t    gOptionR = kFALSE;
+Bool_t    gOptionK = kFALSE;
+TH2F     *gH2 = nullptr;
+TFile    *gHsimple = nullptr;
+TFile    *gCernstaff = nullptr;
```
Comment (vepadulano) [inline test/stressGraphics.cxx:105]: If we're changing this old program might as well try to correct some of its mistakes, beginning from making all these raw pointers `std::unique_ptr` and removing all instances of `new` and `delete` from the file.

**Review by couet — APPROVED** [review]

**Pull request merged**